client/lib/plans/constants.js

```diff
@@ -113,6 +113,7 @@ export const FEATURE_ADVANCED_SEO_TOOLS = 'advanced-seo-tools';
 export const FEATURE_FREE_THEMES_SIGNUP = 'free-themes-signup';
 export const FEATURE_BACKUP_STORAGE_SPACE_UNLIMITED_SIGNUP = 'unlimited-backup';
 export const FEATURE_MEMBERSHIPS = 'memberships';
+export const FEATURE_AUTO_LOADING_HOMEPAGE = 'auto-loading-homepage';
```

Comment:
This is a theme feature, not really a plan feature. Given how it's not used anywhere else, should we just keep it inline in `hasAutoLoadingHomepageFeature()`?

Reply:
👍 yeah, make sense since there isn't a themes/constants.js file either.